src/pages/ContentfulPage.vue

```diff
@@ -205,23 +205,29 @@ export default {
 		};
 	},
 	metaInfo() {
-		return {
-			title: this.title,
-			meta: [
+		const metaObj = { title: this.title.replace('| Kiva', '') };
+
+		if (this.description) {
```

Comment:
In what situations do we not have `description` and/or `canonicalUrl`?

Reply:
Most of the time there won't be a custom canonical URL, and we're still in the process of creating custom descriptions, so we have many pages at the moment that don't have one.